sources/web/datalab/polymer/test/item-list-test.ts

```diff
@@ -436,5 +436,80 @@ describe('<item-list>', () => {
       assert(fifthDetailsContainer.getAttribute('hidden') != null,
           'fifth details container should be hidden');
     });
+
+    it('should hide the filter box by default', () => {
```

Comment:
Are we using 'should' or not in our test descriptions? I guess I would be happy with not using it so our descriptions are a bit shorter.

Reply:
Done.